crates/router/src/core/payments/operations/payment_confirm.rs

```diff
@@ -446,6 +446,21 @@ impl<F: Send + Clone, Ctx: PaymentMethodRetrieve>
         )
         .await?;
 
+        let additional_pm_data = request
+            .payment_method_data
+            .as_ref()
+            .async_map(|payment_method_data| async {
+                helpers::get_additional_payment_data(payment_method_data, &*state.store).await
+            })
+            .await;
```

Comment:
If possible, fetch this in parallel using `tokio::spawn`

Reply:
It seems that I need to clone `state.store` for this to work. Since I am unable to pass this `state.store` by dereferencing because of lifetime problems.

Reply:
You can clone the state and make this work